.travis.yml

```diff
@@ -124,15 +84,7 @@ addons:
 script:
     - make clean_travis
     - make package
-    - >
-      if [[ "$CC_MODULE" = "analyzer" ]]; then
-        make -C analyzer test_unit test_functional test_tu_collector;
-        if [[ "$TRAVIS_OS_NAME" = "linux" ]]; then
-          make -C analyzer test_build_logger
-        fi
-      elif [[ ! -z "$DATABASE" ]]; then
-        make -C web test_matrix_${DATABASE};
-      else
-        make test;
-      fi
+    - make -C analyzer test_unit test_functional
+    - if [[ "$TRAVIS_OS_NAME" = "linux" ]]; then make -C analyzer test_tu_collector test_build_logger; fi
```

Comment:
does the `tu_collector` not work on osx? It should collect for clang compilation commands the required files too.

Reply:
For some reason it doesn't work on osx. I tried to debug it in #1969 but unfortunately without success. I will give it a try later.

Reply:
Ok, I've created a separate issue to fix it #1984